src/Sentry.Extensions.AI/SentryAIConstants.cs

```diff
@@ -0,0 +1,67 @@
+namespace Sentry.Extensions.AI;
+
+internal static class SentryAIConstants
+{
+    /// <summary>
+    /// The list of strings which FunctionInvokingChatClient(FICC) uses to start the tool call <see cref="Activity"/>.
+    /// </summary>
+    internal static readonly string[] FICCActivityNames = ["orchestrate_tools", "FunctionInvokingChatClient.GetResponseAsync", "FunctionInvokingChatClient"];
+
+    /// <summary>
+    /// The string we use to identify our <see cref="ActivitySource"/>.
+    /// </summary>
+    internal const string SentryActivitySourceName = "Sentry.AgentMonitoring";
+
+    /// <summary>
+    /// The string we use to retrieve the span from the <see cref="Activity"/> using a Fused property
+    /// </summary>
+    internal const string SentryActivitySpanAttributeName = "SentryCurrSpan";
```

Comment:
Reading [the code](https://github.com/getsentry/sentry-dotnet/blob/9b72f4c545652c2a9a97ac1ebb3ab8443f6035e6/src/Sentry.Extensions.AI/SentryAIActivityListener.cs#L19-L29), it looks like this is used to store a reference to the span representing the FunctionInvokingChatClient... in which case I think we could give this a more specific name to make that clear.

```suggestion
    internal const string SentryFICCSpanAttributeName = "SentryFICCSpan";
```

Reply:
Fixed in https://github.com/getsentry/sentry-dotnet/pull/4657/commits/b4dc1687fd5a95a6334eca0f9b793ef671c7fe2e